src/components/ui/InputFile.tsx

```diff
@@ -1,19 +1,44 @@
+'use client';
+
 import { useState } from 'react';
 
 import { TextField } from '@mui/material';
+import { createClient } from '@supabase/supabase-js';
 
 const InputFile = () => {
+  const supabaseUrl =
+    process.env.NEXT_PUBLIC_SUPABASE_URL ||
+    'https://acms-backend-2023.onrender.com';
+  const supabaseKey =
+    process.env.NEXT_PUBLIC_SUPABASE_ANON_KEY ||
+    'eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJzdXBhYmFzZSIsInJlZiI6ImZqcWxveHB5a25xY2NyZXR6b3l0Iiwicm9sZSI6ImFub24iLCJpYXQiOjE3MDIzMDI1NjgsImV4cCI6MjAxNzg3ODU2OH0.s4upzMGDuRJ4l-kRK0HMCB6_iSy1ZKATYnzBW2dnoWA';
+  const supabase = createClient(supabaseUrl, supabaseKey);
   const [selectedFile, setSelectedFile] = useState<File | null>(null);
 
-  const handleChange = (event: any) => {
+  const handleChange = async (event: any) => {
     setSelectedFile(event.target.files[0]);
+    try {
+      const {} = await supabase.storage.createBucket('payments');
```

Comment:
usually di mo dito ilagay yung implementation ng kanyang onChange fatt kai para reusable sya na component. Ang usually ginagawa is ipass mo yung handleChange as parameters

Reply:
pwede mo rin ilabas yung selectedFile na state

Reply:
sige notedd.